setup.py

```diff
@@ -108,7 +108,7 @@ def read_version_py(infname):
 
     "six >= 1.10.0",
 
-    # for 'tahoe invite' and 'tahoe join'
+    # For 'tahoe invite' and 'tahoe join'
```

Comment:
This is for busting the cache (`CACHE_KEY` includes `{{ checksum \"setup.py\" }}`)